en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -59,329 +59,192 @@ Let's see how to create a custom sequence using the WSO2 API Manager Tooling Pl
     <tr class="even">
-    <td>New Property Name</td>
-    <td>REST_URL_POSTFIX</td>
+    <td>Property Name</td>
+    <td>requestTime</td>
     </tr>
     <tr class="odd">
-    <td>Value Type</td>
+    <td>Property Type</td>
     <td>Expression</td>
     </tr>
     <tr class="even">
-    <td>Value Expression</td>
-    <td>get-property('YQL')</td>
-    </tr>
-    <tr class="odd">
-    <td>Property Scope</td>
+    <td>Value/Expression</td>
     <td><div class="content-wrapper">
-    <p>Axis2</p>
-        !!! note
-        <p>Since this is a transport level Property you need to set the Property Scope as <strong>Axis2</strong> .</p>
-
+    <p>get-property('api.ut.requestTime')</p>
     </div></td>
     </tr>
     </tbody>
     </table>
 
-    ![](../../../assets/img/Learn/eclipse-property-name.png)
+    ![](../../../assets/img/Learn/eclipse-log-mediator.png)
 
-15. Navigate to the **File** menu, and click **Save** to save the sequence.
-16. Right-click on the sequence and click **Commit File** to push the changes to the Publisher server.
+13. Navigate to the **File** menu, and click **Save** to save the sequence.
 
-        !!! note
+    !!! PizzaOrderInSeq
+        ``` java
+        <?xml version="1.0" encoding="UTF-8"?>
+        <sequence name="PizzaOrderInSeq" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+        <log description="" level="custom" separator=",">
+           <property name="request'path" value="get-property('REST_FULL_REQUEST_PATH')"/>
+           <property name="requestTime" value="get-property('api.ut.requestTime')"/>
+        </log>
+        </sequence>
+        ```
+
+14. Right-click on the sequence and click **Commit File** to push the changes to the Publisher server.
+
+!!! note
     Alternatively, you can create a CAR file including the sequences and can deploy it in API Manager. For more information, see [Deploying Composite Applications in the Server](https://docs.wso2.com/display/ADMIN44x/Deploying+Composite+Applications+in+the+Server) , which is in the WSO2 Administration Guide.
 
 
-17. Sign in to the API Publisher again, select the API that you created earlier.
-    ![](../../../assets/img/Learn/weatherapi-overview.png)
-18. Go to the API's **Runtime Configurations** tab, enable the **Message Mediation** switch and enable **Custom Policies**. Now you can upload the sequence that you created for the In flow. Next, click **Save** to engage the sequence to the API.
+15. Sign in to the API Publisher again, select the API that you created earlier.
 
-        !!! tip
-    **Tip** : It might take a few minutes for the sequence to be uploaded into the API Publisher. If it isn't there, please check again later.
+    ![](../../../assets/img/Learn/pizzashack-overview.png)
 
+16. Go to the API's **Runtime Configurations** tab, enable the **Message Mediation** switch and enable **Common Policies**. Now you can select the sequence that you created for the In flow. Next, click **Save** to engage the sequence to the API.
```

Comment:
Enable  **Message Mediation**  and **Common Policies**. Break these into steps.
 Select the sequence created previously for the In flow.
Next, click **Save** to engage the sequence to the API